Externals.cfg

```diff
@@ -38,8 +38,8 @@ required = True
 protocol = git
 repo_url = https://github.com/NOAA-GSL/EMC_post
 # Specify either a branch name or a hash but not both.
-#branch = RRFS_dev
-hash = 9b9c55c
+branch = RRFS_dev
+#hash = 9b9c55c
```

Comment:
Please use a particular hash for the code that you would like to use. Using the top of a branch for workflows running in real time is hard to reconstruct for future use if we need to rebuild.